modules/phase_field/src/postprocessors/GrainBoundaryArea.C

```diff
@@ -0,0 +1,49 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "GrainBoundaryArea.h"
+#include "MooseUtils.h"
+
+template<>
+InputParameters validParams<GrainBoundaryArea>()
+{
+  InputParameters params = validParams<ElementIntegralPostprocessor>();
+  params.addRequiredCoupledVarWithAutoBuild("v", "var_name_base", "op_num", "Array of coupled variables");
+  params.addParam<Real>("grains_per_side", 2.0, "Number of grains sharing a boundary. This should be 2.0 in bulk crystals and 1.0 for dispersed particles.");
```

Comment:
How about "number of order parameters contacting a boundary"  and "should be 2.0 in polycrystals".